contrib/elasticsearch/readme.md

```diff
@@ -0,0 +1,56 @@
+
+# ELK Samples
+
+Helpful setup guide: https://www.elastic.co/guide/en/kibana/8.3/docker.html
+
+Note: These commands are meant to be run from the root of the Emissary project 
+
+## Elasticsearch and Kibana 
+
+Run Elasticsearch:
+```shell
+docker network create elastic
+docker run --name elasticsearch --hostname elasticsearch --net elastic -p 9200:9200 -p 9300:9300 -t docker.elastic.co/elasticsearch/elasticsearch:8.3.2
+```
+
+Copy the generated password and enrollment token and save them in a secure location. These values are shown only when you start Elasticsearch for the first time. 
+You’ll use these to enroll Kibana with your Elasticsearch cluster and log in.
+```shell
+export  ESPASSWORD="<generated password>"
+```
+
+Run Kibana:
+```shell
+docker run --name kibana --hostname kibana --net elastic -p 5601:5601 docker.elastic.co/kibana/kibana:8.3.2
+```
+
+Get certs:
+```shell
+mkdir target/tls
+docker cp elasticsearch:/usr/share/elasticsearch/config/certs/http_ca.crt target/tls/
+docker cp elasticsearch:/usr/share/elasticsearch/config/certs/http.p12 target/tls/
+docker exec -it elasticsearch bin/elasticsearch-keystore show xpack.security.http.ssl.keystore.secure_password
+openssl pkcs12 -in target/tls/http.p12 -out target/tls/http.key.pem -nocerts -nodes
```

Comment:
I might be missing something very simple.
What is the password for http.p12? It is not the elastic user password.
Or did you generate your own p12 crts to use?

Reply:
Should be the output of the previous command:
```
docker exec -it elasticsearch bin/elasticsearch-keystore show xpack.security.http.ssl.keystore.secure_password
```

Reply:
I updated it to use an env var. I was having some trouble with the docker output containing a carriage return (it's probably why I did not do it this way). Let me know if that works.

Reply:
Yep it works. Thanks. I missed the previous command.